README.md

```diff
@@ -9,6 +9,8 @@ The Alfresco Digital Business Platform can be deployed to different environments
 - [Deploy to Docker for Desktop - Mac](#docker-for-desktop---mac)
 - [Deploy to Docker for Desktop - Windows](#docker-for-desktop---windows)
 
+*Note:* If using Azure instead of AWS then you will need to use Azure Databases instead of Postgres.
```

Comment:
I'm not sure this is the best spot for this note, and I think I'd make it a bit more general.

Perhaps in the notes section of the AWS section, something along the lines of:
> When deploying to cloud environments like AWS and Azure you should consider using native database services from those providers rather than deploying Postgres within the Kubernetes cluster.

A note is probably also needed somewhere on the README-cfn.md page.